pkgs/misc/uboot/default.nix

```diff
@@ -15,6 +21,7 @@
 , armTrustedFirmwareRK3399
 , armTrustedFirmwareS905
 , buildPackages
+, crustTeresA64
```

Comment:
Can you explain why this change was needed?

This is unused, currently, since it it commented-out. Is this a wish?